src/data/actions/courseInfo.js

```diff
@@ -192,22 +199,40 @@ function handleCourseRuns(dispatch, courseRunData, course, submitReview, interna
       newCourse.course_runs[i] = response.data;
     });
     dispatch(editCourseSuccess(newCourse));
-    if (submitReview) dispatch(courseSubmittingSuccess());
+    if (submitReview) {
+      dispatch(courseSubmittingSuccess());
+    }
+    if (internalReview) {
+      refreshData();
+    }
   }).catch((error) => {
     dispatch(editCourseFail(['Course Run edit failed, please try again or contact support.']
       .concat(getErrorMessages(error))));
     if (submitReview) dispatch(courseSubmittingFailure());
   });
 }
 
-function editCourse(courseData, courseRunData, submittingRunForReview, isInternalReview) {
+function editCourse(
+  courseData,
+  courseRunData,
+  submittingRunForReview,
+  isInternalReview,
+  refreshData,
```

Comment:
Do all of these functions have to be so many lines like this? Is there a way to do it in only 2 or 3 lines?